rust/connlib/tunnel/src/dns.rs

```diff
@@ -27,13 +27,11 @@ const DNS_PORT: u16 = 53;
 
 /// Tells the Client how to reply to a single DNS query
 #[derive(Debug)]
-pub(crate) enum ResolveStrategy<T, U, V> {
+pub(crate) enum ResolveStrategy<'a> {
     /// The query is for a Resource, we have an IP mapped already, and we can respond instantly
```

Comment:
Slightly outdated comment, the "instantly" is now obsolete because we don't defer DNS responses anymore.